src/cascadia/TerminalControl/TermControl.h

```diff
@@ -151,6 +152,9 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         TerminalConnection::ConnectionState ConnectionState() const;
         IControlSettings Settings() const;
 
+        IControlSettings UnfocusedAppearance() const;
+        void UnfocusedAppearance(IControlSettings unfocusedAppearance);

```

Comment:
```suggestion
        GETSET_PROPERTY(IControlSettings, UnfocusedAppearance);
```
Seems like there's nothing special about this one?